CorsixTH/Lua/world.lua

```diff
@@ -1095,7 +1096,7 @@ function World:onEndDay()
   --check if it's time for a VIP visit
   if self.game_date:isSameDay(self.next_vip_date) then
     if #self.rooms > 0 and self.ui.hospital:hasStaffedDesk() then
-      self.hospitals[1]:createVip()
+      local_hospital:createVip()
```

Comment:
And now you hope `self.ui.hospital` and `local_hospital` are actually the same thing.

Reply:
Might need some elaboration on this. Are you suggesting to use ``local_hospital:hasStaffedDesk()`` on 1098 instead?